pyproject.toml

```diff
@@ -33,7 +33,7 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = "^3.8"
```

Comment:
This will reduce the frequency of version mismatch problems when including cosmpy as a dependency.